content/source/docs/enterprise/api/oauth-clients.html.md

```diff
@@ -26,10 +26,12 @@ This endpoint allows you to create a VCS connection between an organization and
 
 ~> **Note:** This endpoint does not currently support creation of a Bitbucket Server OAuth Client.
 
-Result  | Status and response
---------|------------------------
-Success | HTTP 201 and a JSON API document (`type: "oauth-clients"`).
-Failure | HTTP 4XX and a JSON API list of error messages.
+Status  | Response                                | Reason
+--------|-----------------------------------------|----------
+[201][] | [JSON API document][] (`type: "oauth-clients"`) | The request was successful
+[404][] | [JSON API error object][]               | Organization not found or user unauthorized to perform action
```

Comment:
I think this should probably say "entity not found" or "OAuth client not found"

Reply:
Oh, hmm, I was assuming it was specific to the org since the org is in the URL path... but you're probably right that it 404s for any not found.

Reply:
It was "Organization" because of the URL: `POST /organizations/:organization_name/oauth-clients`. Should we change it to "entity"?

Reply:
Oh, this is for creation specifically - no, you're right, I'm confused. :)